### DISCUSSION TITLE
Darkwood on your mobile

### DISCUSSION THREAD

**Discussion by TheMercenary333** [skyline-emu/skyline#1345] [Game Requests]
It would good be great to play Darkwood on mobile. What do you think? Anybody has been tried on the emulator?
![capsule_616x353](https://user-images.githubusercontent.com/74915480/169115719-6fda1049-30d5-4577-978e-0e6b3933c462.jpg)

**Comment by Chibuzo2003** [comment]
Nope I don't even have one game I don't know how to download games is anybody willing to help me
